### DISCUSSION TITLE
Request to add functionality to configure virtual inventory

### DISCUSSION THREAD

**Discussion by EndlessLegends** [Zrips/CMI#5651] [Feature Requests]
Hello!
Questions about adding features.

It is not easy to remember all of the many commands of this plugin.
Also, I want users playing on my server to be able to easily use the convenient features provided by cmi.

I would like to collect commands that can be used by operators or users in a virtual inventory.

Can you add some functionality so I can create a custom gui?

++ I bought your plugin and never regretted it.
++ Thanks to the cmi team for continuous development, bug fixes, feature additions and feedback.

Thank you.

**Comment by CloudeLecaw** [comment]
What about plugins like chestcommands or advancedmenue and many Others that allow you creating guis? You could create a gui that executes the commands with these plugins.
